### ISSUE TITLE
Demo + bug: unpaired brain MR pre-trained model performance poor

### ISSUE THREAD

**Issue by NMontanaBrown** [DeepRegNet/DeepReg#361]
## Subject of the issue

The provided pre-trained network doesn't seem to warp the images fed in as provided in the demo.

If the documentation request is approved, would you be willing to submit a PR?

**Yes** / No

## Your environment

- DeepReg version (commit hash or tag) v. 0.1.6-alpha

- OS (e.g. Ubuntu 20.04, MacOS 10.15, etc.) Mac High Sierra

- Python Version (3.7, 3.8, etc.)   3.7

- TensorFlow 2.3

## Steps to reproduce

Run the demo as indicated for unpaired brain mri, and check the visualisation.

## Expected behaviour

We would expect the pre-trained registration network to warp the moving image to resemble the fixed image. 

## Actual behaviour

As in Issue #347 the performance of the pre-trained model is strange - the pred-fixed-image (the warped moving image) and labels are not warped (appear the same), if we look at the visualisation.

##  Suggested fix

Train a model for longer...?

**Comment by acasamitjana** [comment]
I have now tried different hyperparameters and trained longer and the performance has improved. I am sharing a video example: https://liveuclac-my.sharepoint.com/:v:/g/personal/rmapcas_ucl_ac_uk/EaHqDbzAOAxKrPcsXAxnhkoBhuoGTs2NmYgAJ6K77XkLDQ?e=UgIuBc

**Comment by YipengHu** [comment]
Nice - need a gallery for these

**Comment by mathpluscode** [comment]
> I have now tried different hyperparameters and trained longer and the performance has improved. I am sharing a video example: https://liveuclac-my.sharepoint.com/:v:/g/personal/rmapcas_ucl_ac_uk/EaHqDbzAOAxKrPcsXAxnhkoBhuoGTs2NmYgAJ6K77XkLDQ?e=UgIuBc

How you made this video? I feel like we can add this feature;)

**Comment by NMontanaBrown** [comment]
> > I have now tried different hyperparameters and trained longer and the performance has improved. I am sharing a video example: [https://liveuclac-my.sharepoint.com/✌️/g/personal/rmapcas_ucl_ac_uk/EaHqDbzAOAxKrPcsXAxnhkoBhuoGTs2NmYgAJ6K77XkLDQ?e=UgIuBc](https://liveuclac-my.sharepoint.com/:v:/g/personal/rmapcas_ucl_ac_uk/EaHqDbzAOAxKrPcsXAxnhkoBhuoGTs2NmYgAJ6K77XkLDQ?e=UgIuBc)
> 
> How you made this video? I feel like we can add this feature;)

Honestly considering predict outputs all the images I don't see what's stopping us from writing a small script to make videos/gifs to visualise as a util for the package - simple file io...? I used some online gif generator but programatically easy?

Also @YipengHu I can send the video/gif for lung ct I made for the MEC and we can put it somewhere too?

**Comment by acasamitjana** [comment]
I close this ticket since it's been improved by PR  #422
